examples/quarkus-jib-extension/README.md

```diff
@@ -0,0 +1,100 @@
+# getting-started Project + Quarkus jib extension
+
+This project uses Quarkus, the Supersonic Subatomic Java Framework.
+Quarkus Jib documentation: https://quarkus.io/guides/container-image .
+
+If you want to learn more about Quarkus, please visit its website: https://quarkus.io/ .
+
+## Running the application in dev mode
```

Comment:
This is helpful information! But it might be more valuable to keep this README focused on what needs to be done on the Jib side of things.